Project.toml

```diff
@@ -4,16 +4,19 @@ authors = ["Bernard Brenyah", "Andrey Oskin"]
 version = "0.1.0"
 
 [deps]
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
```

Comment:
So you do depend on Distances?

Reply:
> So you do depend on Distances?

Yes, Distances is needed to make the transform function but not originally depended on with regards to the package itself.